update-cmdlet-documentation.ps1

```diff
@@ -95,7 +95,7 @@ $newOrUpdatedFiles = @(
     Update-MarkdownHelp -Path $documentationPath -ExcludeDontShow | Where-Object BaseName -ne $NewCommand
 )
 
-$incompleteFiles = $newOrUpdatedFiles | Select-String '\{\{[^}]+}}' | Select-Object -ExpandProperty Path
+$incompleteFiles = $newOrUpdatedFiles | Select-String '\{\{[^}]+}}' | Select-Object -ExpandProperty Path -Unique

```

Comment:
Nothing that needs to be actioned (other than resolving this comment before merge). This is the commit message for this change to save further reviewers from having the same question I did:

>  (maint) Cmdlet docs - remove duplicate path warns
> 
> When a cmdlet doc is generated / has template tokens from PlatyPS in it,
> we warn the user and output the path to the relevant file.
> 
> This change ensures we only do that once per file.